kernel: bump 5.15 to 5.15.112

graysky2 opened the pull request:
Manually adjusted before running quilt due to new location in tree:
	backport-5.15/780-v5.16-bus-mhi-pci_generic-Introduce-Sierra-EM919X-support.patch
	backport-5.15/781-v6.1-bus-mhi-host-always-print-detected-modem-name.patch
	pending-5.15/790-bus-mhi-core-add-SBL-state-callback.patch

All other patches automatically rebased.

Build system: x86_64
Build-tested: bcm2711/RPi4B, ramips/tplink_archer-a6-v3, filogic/xiaomi_redmi-router-ax6000-ubootmod
Run-tested: bcm2711/RPi4B, ramips/tplink_archer-a6-v3, filogic/xiaomi_redmi-router-ax6000-ubootmod

hauke commented on target/linux/generic/backport-5.15/781-v6.1-bus-mhi-host-always-print-detected-modem-name.patch:
```diff
@@ -21,11 +21,11 @@ Link: https://lore.kernel.org/r/20220831100349.1488762-1-koen.vandeputte@citymes
---- a/drivers/bus/mhi/pci_generic.c
-+++ b/drivers/bus/mhi/pci_generic.c
+--- a/drivers/bus/mhi/host/pci_generic.c
++++ b/drivers/bus/mhi/host/pci_generic.c
```
Please adapt the file `package/kernel/linux/modules/other.mk` too.

graysky2 replied:
I did this in the 1st commit of this PR.  Would you like them squashed into a single commit?
https://github.com/openwrt/openwrt/pull/12641/commits/58bb9d13c89db83b9d449fb428db511cbcd8eef8

graysky2 committed:
kernel: other: adjust paths for new location

This upstream commit[1] moves drivers/bus/mhi/core/ to
drivers/bus/mhi/host/ so we need to correct for that here.

https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v5.15.112&id=9fbf10148f31a63b6d8c2b0e56cb97eb173d600b

Signed-off-by: John Audia <therealgraysky@proton.me>

graysky2 committed:
kernel: bump 5.15 to 5.15.112

Manually adjusted before running quilt due to new location in tree:
	backport-5.15/780-v5.16-bus-mhi-pci_generic-Introduce-Sierra-EM919X-support.patch
	backport-5.15/781-v6.1-bus-mhi-host-always-print-detected-modem-name.patch
	pending-5.15/790-bus-mhi-core-add-SBL-state-callback.patch

All other patches automatically rebased.

Build system: x86_64
Build-tested: bcm2711/RPi4B, ramips/tplink_archer-a6-v3, filogic/xiaomi_redmi-router-ax6000-ubootmod
Run-tested: bcm2711/RPi4B, ramips/tplink_archer-a6-v3, filogic/xiaomi_redmi-router-ax6000-ubootmod

Signed-off-by: John Audia <therealgraysky@proton.me>

PolynomialDivision commented:
Run-tested TOTOLINK A7000R. Looks good to me.

PolynomialDivision commented:
Thanks! Merged:
- https://github.com/openwrt/openwrt/commit/b2c0d9d01f02f58965eae7d14104423bb3510bd5
- https://github.com/openwrt/openwrt/commit/6605b595eda76533a9e6fb907f80ede3d1b255d1
